opencti-platform/opencti-front/src/private/components/settings/users/UserConfidenceLevel.tsx

```diff
@@ -24,7 +24,17 @@ const UserConfidenceLevel: React.FC<UserConfidenceLevelProps> = ({ user }) => {
     );
   }
 
-  // TODO: add overrides in a tooltip when in use
+  const renderOverrides = () => {
+    return (
+      <>
+        {user.effective_confidence_level?.overrides?.map((override, index) => (
+          <div key={index} style={{ marginTop: '5px' }}>
+            {`${override.entity_type}: ${override.max_confidence}`}
+          </div>
+        ))}
+      </>
+    );
+  };
```

Comment:
It's more React oriented to created a component instead of creating some "render" functions.
Here something like `ConfidenceOverrides`

Reply:
Same for `renderSource` below